client/src/Homes.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './style.css';
+
+const Homes = (props) => {
+  const { data } = props;
+  const listing = data.map((home) => {
+    return (
```

Comment:
As an initial thought--it may be worth thinking about how some of your classNames may/may not clash with others classNames when combined, and if you want to have a naming convention specific to your module!